docs/self-managed/operational-guides/update-guide/820-to-830.md

```diff
@@ -9,8 +9,7 @@ description: "Review which adjustments must be made to migrate from Camunda 8.2.
 The following sections explain which adjustments must be made to migrate from Camunda 8.2.x to 8.3.0 for each component.
 
 :::caution
-Updating Operate, Tasklist, and Optimize from 8.2.x to 8.3.0 will potentially take longer than expected, depending on the data to be migrated.
-Additionally, we identified some bugs that could also prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
+Updating Operate, Tasklist, and Optimize from 8.2.x to 8.3.x will potentially take longer than expected, depending on the data to be migrated. We identified some bugs in the 8.3.0 release that could prevent migration from succeeding, which are fixed in 8.3.1. When updating, it is therefore recommended you skip version 8.3.0 and update directly to 8.3.1.
```

Comment:
Let's use the same for next and 8.3 for now, and update both as well once the patches are out to have it consistent. wdyt?
```suggestion
We identified some bugs in 8.3.0 that could prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
```